src/styles/layout/_layout.scss

```diff
@@ -93,6 +93,23 @@ main
   width: space(12x);
 }
 
+.docs-layout-minimal
```

Comment:
We're using alpha, beta, gamma for layout abstraction in AEM. We could consider figuring out a naming convention to use as a recommendation.